packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -109,19 +109,27 @@ contract L2Genesis is Deployer {
         });
     }
 
-    /// @dev Sets the precompiles, proxies, and the implementation accounts to be `vm.dumpState`
-    ///      to generate a L2 genesis alloc.
     /// @notice The alloc object is sorted numerically by address.
+    ///         dev Sets the precompiles, proxies, and the implementation accounts to be `vm.dumpState`
+    ///         to generate a L2 genesis alloc.
```

Comment:
Is the remaining `dev` without `@` intentional? Not sure what semgrep wants.

Reply:
this is a typo